sre-recipes/sandboxctl

```diff
@@ -41,11 +41,13 @@ def get_valid_recipes(attributes):
     """
 
     recipe_objs = {}
-    for recipe_num, (attribute_name, attribute) in enumerate(attributes):
+    recipe_num = 0
+    for (attribute_name, attribute) in attributes:
         if isclass(attribute) and attribute is not Recipe and issubclass(attribute, Recipe):
             try:
                 recipe_obj = attribute()
                 recipe_objs[f"recipe{recipe_num}"] = recipe_obj
+                recipe_num += 1
```

Comment:
I do kind of question using index numbers as recipe identifiers though. Are we sure that this sorting will always be stable? Isn't it possible that previous identifiers change if you add new recipes?

Reply:
I was thinking the same earlier - and I think at the moment how it is configured is alphabetical in a sense so if someone wrote a "a_recipe" that recipe would then become recipe0 and then the rest of the recipes would shift accordingly...

Reply:
It looks like it should be pretty easy to add a `get_name` method or attribute to all recipes, and grab it with `recipe_obj.get_name()`

Reply:
Gotcha - I gave them all a name now, this is what the help command looks like:
```
Usage: sandboxctl [OPTIONS] [break|restore|verify|hint] [currency_recipe|encoding_recipe]
```